browser/src/map/Clipboard.js

```diff
@@ -866,15 +866,15 @@ L.Clipboard = L.Class.extend({
 	},
 
 	// Gets status of a copy/paste command from the remote Kit
-        _onCommandResult: function(e) {
-                if (e.commandName === '.uno:Copy' || e.commandName === '.uno:Cut')
+    _onCommandResult: function(e) {
```

Comment:
Nit: I think the indent of this line used to match with the `}` at the end of the previous function, and now it doesn't do that anymore.